extensions/libraries/redcore/form/base.php

```diff
@@ -73,80 +74,6 @@ public static function getInstance($name, $data = null, $options = array(), $rep
 	}
 
 	/**
-	 * Override the validate() method to store the error per field.
-	 *
-	 * @param   array   $data   An array of field values to validate.
-	 * @param   string  $group  The optional dot-separated form group path on which to filter the
-	 *                          fields to be validated.
-	 *
-	 * @return  mixed  True on sucess.
-	 */
-	public function validate($data, $group = null)
```

Comment:
Errors returned are grouped by field so instead of having all the exceptions in an array you get them grouped by field.

That's useful for instance for ajax requests because you can send the full form when a field is changed and then show errors.

See how is it used on mipayway:
https://bitbucket.org/redwebdk/mipayway/src/9d592a603a8339f0d2adae8b846277456348afbc/extensions/libraries/mipayway/form/ajax_validator.php?at=master&fileviewer=file-view-default#ajax_validator.php-24

This update would break that